openbook_notifications/migrations/0013_auto_20190709_1919.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.2 on 2019-07-09 17:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openbook_notifications', '0012_auto_20190623_1657'),
+    ]
+
```

Comment:
Not sure why this migrations showed up, these are not my changes 🤔

Reply:
huh what

Reply:
don't commit it then 😬

Reply:
Yea it was by mistake :( cos i added files in the `openbook_notifications`